src/helpers/utils.ts

```diff
@@ -182,3 +183,7 @@ export function isSnapshotUrl(url: string) {
 
   return false;
 }
+
+export function toChecksumAddress(address: string) {
+  return getAddress(address.toLowerCase());
+}
```

Comment:
```suggestion

export function toChecksumAddress(address: string) {
  try {
    return getAddress(address);
  } catch (e) {
    return address;
  }
}
```

Reply:
Why do you need the `.toLowerCase()`

Reply:
If you pass a mixed case address into `getAddress`, it throws. So if a user for some reason enters a mixed case safe address when they ad a treasury, we can safely convert it to checksum by making it all lowercase first so it doesn't throw. 

This is all really just to make safe's API happy.